wire-compiler/src/main/java/com/squareup/wire/WireCompiler.java

```diff
@@ -1298,9 +1298,17 @@ private void compileWithJavaPoet() throws WireException {
         continue; // Don't emit anything for files not explicitly compiled.
       }
 
+      TypeWriter typeWriter = new TypeWriter(javaGenerator, options.emitOptions);
```

Comment:
I giggle internally at the juxtaposition this class name evokes every time I see it. Especially since it's not done obnoxiously like making a calculator implementation in `Abacus.java` or something.